scripts/lint-style.py

```diff
@@ -14,9 +15,13 @@
 
 exceptions = []
 
-with open("scripts/style-exceptions.txt", encoding="utf-8") as f:
-    for line in f.readlines():
+SCRIPTS_DIR = Path(__file__).parent
+ROOT_DIR = SCRIPTS_DIR.parent
```

Comment:
How about just `os.chdir(ROOT_DIR)`?  This should be the least error-prone, and doesn't complicate the rest of the script.

Reply:
Happy to do that instead, though doing it that way still either would require a bunch of the below or otherwise have some slightly odd behavior -- specifically if you pass an absolute path (to the file to lint now), you'll get e.g. the notation check comparing to a relative path, so you'd have to pass relative paths to the root of the mathlib repo even if you weren't alongside it.

Reply:
Ok, then having relative paths everywhere is probably the lesser evil.